### ISSUE TITLE
During loading image, multiple key repeats might happen.

### ISSUE THREAD

**Issue by freed00m** [eXeC64/imv#272]
Hi,

Could you provide blocking for key repeats during image load finish?
The most obvious when this happens is when trashing files.

```
[binds]
<Shift+D> = exec gio trash "$imv_current_file"; close
``` 

When I trash the file the next one loads in imv and then there is small lag, if I take my time with key press the binding get's stacked up and repeats so I might end up deleting multiple files even though I pressed it once.

**Comment by JustinLovinger** [comment]
This can also happen after image has loaded. I made a binding that spawns a new terminal in the background to edit the file name. Sometimes, it will bomb the computer with infinite terminals until the system crashes. I think what is happening is that opening the terminal causes a moment of lag, which causes a key-repeat, which opens more terminals, which causes more lag, etc.

Also possibly related: if I don't spawn the terminal in the background, imv will keep opening new terminals when the old one closes. I suspect it is the same key repeat issue, but with a blocking call between each repeat.

**Comment by eXeC64** [comment]
This has been fixed now I believe, same issue as #201
